Downloads/WeatherApplication/app/src/main/java/com/example/weatherapplication/model/Forecast.kt

```diff
@@ -0,0 +1,3 @@
+package com.example.weatherapplication.model
+
+data class Forecast(val date: Long, val sunrise: Long, val sunset: Long, val temp: ForecastTemp, val pressure: Float, val humidity: Int)
```

Comment:
This class should be called `DayForecast` as specified in the Assignment 3 specification.